### PR TITLE
Rename trace exporters that have inconsistent naming.

### PR DESCRIPTION
**Pull request by HailongWen**
This PR contains two commits: 
1. rename old files 
2. add old files back. I didn't use one to implement another, just to avoid adding new logic.

The reason for this two-step renaming is to preserve history on the renamed files (old files will be deprecated and removed some day in the future).
Although the calculated diff shows that `**TraceExporter` are added files, they are just moved and renamed. Using `git blame` may give the correct change history.

### CODE REVIEW

**Comment by bogdandrutu** [comment]
Why did you not rename the classes and add a deprecated class with the old names that just calls into the new class?

**Comment by HailongWen** [comment]
I didn't make the old names call into the new class to minimize manual code. I can do that.

**Comment by HailongWen** [comment]
@bogdandrutu PTAL.

File: exporters/trace/logging/src/test/java/io/opencensus/exporter/trace/logging/LoggingExporterTest.java [hunk 1]
```diff
@@ -20,7 +20,7 @@
 import static org.mockito.Matchers.eq;
 import static org.mockito.Mockito.verify;
 
-import io.opencensus.exporter.trace.logging.LoggingExporter.LoggingExporterHandler;
```
Comment (bogdandrutu) [inline exporters/trace/logging/src/test/java/io/opencensus/exporter/trace/logging/LoggingExporterTest.java]: This file can be removed.

File: exporters/trace/stackdriver/src/test/java/io/opencensus/exporter/trace/stackdriver/StackdriverExporterTest.java [hunk 2]
```diff
@@ -45,9 +45,9 @@ public void registerUnregisterStackdriverExporter() {
     StackdriverExporter.register(spanExporter, handler);
     verify(spanExporter)
         .registerHandler(
-            eq("io.opencensus.exporter.trace.stackdriver.StackdriverExporter"), same(handler));
```
Comment (bogdandrutu) [inline exporters/trace/stackdriver/src/test/java/io/opencensus/exporter/trace/stackdriver/StackdriverExporterTest.java]: Same thing. You can remove this file.

**Comment by HailongWen** [comment]
fixes #925.

**Pull request merged**